contents/blog/brand.md

```diff
@@ -0,0 +1,152 @@
+* How much you invest in each function in your company relative to each other.
+* What you look for when you hire people, and it's the process you use to hire with.
+* How you decide what to build – do you ship features early, or do you focus on meeting the needs of your largest customer with every new thing?
+* How much you are design vs engineering vs product vs sales vs marketing-led – how many people are in each function, but also your processes for deciding what to work done.
+* The frequency and style with which you email your customers, for marketing, product updates or your activation flow.
+* The kind of people you hire into customer success or sales.
+* The background experience of those in your marketing team.
+* The tone of voice on your website.
+* Who is on your board.
+* How your pricing works.
+* How your social media works.
+* How helpful you are to random founders that message you.
+* The marketing channels you pick.
+* What you talk about in your blog. I'm writing this piece for brand reasons (transparency / equipping developers with knowledge to build successful products), nothing else. I don't care or try to generate conversions from this post, I just want to leave an impression on people. And it's who I am – I'm writing this to pass the time on a long plane journey.
+
+## You can't 80:20 everything
+
+A lot of design or brand work has an _increasing_ marginal return. This is because a great brand's purpose is that you stand out. You do _not_ stand out by being average!
+
+So, if you're serious, go nuts on your brand. Go weirdly hard.
+
+To give you an example, we built a [merch store](/merch):
+
+* We started with me emailing users for their address and size, ordering merch and having it delivered to my house, and me then posting it internationally to them. [Do things that don't scale](https://paulgraham.com/ds.html)!
+
+* Next, we set up a store on Shopify. I just threw our logo onto an amusingly wide selection of items – PostHog shower curtain, anyone? It was cool to just have merch for sale and a way of giving it.
+
+* Then we created custom components and built it into our website directly, so it's perfectly integrated.
+
+* Then we booked a photo shoot and took custom photos of our team wearing everything.
+
+* Now we're working on limited editions of certain items to create collectible series of t-shirts
+
+Who knows what happens next!
+
+This single thing is easy enough to copy, but it's just one activity among hundreds that builds a brand.
+
+## Doesn't this imply doing everything to a really high standard and a ridiculous workload?
+
+Yes. You can be iterative with it, though. Not everything was amazing on day one, but we keep improving and tweaking everything over time.
+
+So, get out there, and be bold
+
+If your brand isn't going to turn off some people that _might_ have bought your product, then it isn't strong enough to stand out.
+
+Engage your _best_ customers with your brand, and put off your _worst_ ones.
```

Comment:
personal preference but i don't think the italics work here. you wouldn't put that much emphasis on 'best' and 'worst' if you were saying this out loud so it makes it scan weird. i'd remove 'em, but up to you